README.md

```diff
@@ -37,5 +37,22 @@ If we want to expand the API for connections, we can always add more symbols
 for parsing. One I was thinking about is `\->`, which would denote an
 "ending" connection (to signal destroying the path to reach the current action)
 
+# Game objects
 
+1) Know the attributes of your object (duh)
 
+2) Go to **builtin.py** -> **Obj** and find **Obj** class. This is where you initialize the object. **NOTE:** You don't assume the player is going to use an object. This is merely initialization. Tip: Initialize the logic of your object here. 
+
+3) The hard part is done - go nuts and build your own class definition.
+
+4) Do not forget to go to **crpg.py** -> **generate()** iff you are:
+    - creating a new object type. 
+        - go to the first for loop and look for the first if statement. 
+        - remember to include your new object type in n_types
```

Comment:
I think we should steer away from instructions on modifying the code itself. Instead, we can comment above and within the functions we write so that devs can understand exactly what each function does!